### ISSUE TITLE
xlwings.Sheet('Sheet').print ?

### ISSUE THREAD

**Issue by AshleyJackson** [xlwings/xlwings#504]
Is it possible to currently print a sheet? If not, can I get a feature request?

**Comment by fzumstein** [comment]
It's currently not baked in, but for now you can use the underlying libs to achieve what you want, see http://docs.xlwings.org/en/stable/missing_features.html

In your case, something like `xw.Sheet('Sheet').xl_sheet.PrintOut()` should do the trick

**Comment by AshleyJackson** [comment]
Thanks for the reply.

Does the .PrintOut() method allow you to control the amount of copies that are printed out?

**Comment by fzumstein** [comment]
yes: https://msdn.microsoft.com/en-us/library/office/ff838253.aspx

**Comment by AshleyJackson** [comment]
The syntax for this is confusing me. I currently have it as:

`xlwings.Sheet("Invoice").xl_sheet.PrintOut("Copies:=2")
`
and it doesn't work. I have also tried:

`xlwings.Sheet("Invoice").xl_sheet.PrintOut(), "Copies:=2"
`

When using the top method I receive this error:

`pywintypes.com_error: (-2147352567, 'Exception occurred.', (0, u'Microsoft Excel', u'PrintOut method of Worksheet class failed', u'xlmain11.chm', 0, -2146827284), None)
`

When running the bottom script, it actually prints, but only prints one copy. Not the specified amount.

**Comment by fzumstein** [comment]
try `PrintOut(Copies=2)`
Am 06.07.2016 1:31 vorm. schrieb "Ashley Jackson" <notifications@github.com

**Comment by AshleyJackson** [comment]
That didn't work. It still printed out one copy.

**Comment by fzumstein** [comment]
Looks like a pywin32 bug to me, `xlwings.Sheet("Invoice").xl_sheet.PrintOut(Copies=2)` should work. What about just doing the following as a workaround?

```
copies = 2
for i in range(copies):
    xlwings.Sheet("Invoice").xl_sheet.PrintOut()
```

**Comment by AshleyJackson** [comment]
That workaround works for now. Thanks.

**Comment by chrisluedtke** [comment]
I would also like to request a print feature.

When I try ` xw.Sheet("Sheet").xl_sheet.PrintOut()` 
I get `AttributeError: 'Sheet' object has no attribute 'xl_sheet'`

When I enter `sht.api` 
I get `<xlwings._xlwindows.COMRetryObjectWrapper at 0x16dcd1e4f98>`

Using Jupyter notebook.

**Comment by fzumstein** [comment]
`xl_sheet` is gone since 0.9.0.

`sht.api.PrintOut()` should work, the output you see is expected.

**Comment by chrisluedtke** [comment]
Thank you! I did try to ctrl-F the "What's new" page for "xl_sheet" but there were no hits.

**Comment by stevenvandorpe** [comment]
> Looks like a pywin32 bug to me, `xlwings.Sheet("Invoice").xl_sheet.PrintOut(Copies=2)` should work. What about just doing the following as a workaround?
> 
> ```
> copies = 2
> for i in range(copies):
>     xlwings.Sheet("Invoice").xl_sheet.PrintOut()
> ```

I have found this is actually a problem for _all_ parameters passed to PrintOut(). Is there a way to fix this?
